http-server-netty/src/test/groovy/io/micronaut/http/server/netty/handler/PipeliningServerHandlerSpec.groovy

```diff
@@ -354,6 +361,161 @@ class PipeliningServerHandlerSpec extends Specification {
         ch.checkException()
     }
 
+    def 'decompression parts to full'(ChannelHandler compressor, CharSequence contentEncoding) {
```

Comment:
Wouldn't it be better to write these tests using ordinary controller-client tests? That would be more understandable for people like me who are not that familiar with the netty and it might be better if you do some refactoring/Netty upgrade.

Reply:
yea i can add some normal controller tests

Reply:
added one though it still uses netty for the client compression part